x-pack/plugins/ml/server/models/job_service/index.ts

```diff
@@ -13,11 +13,16 @@ import { newJobCapsProvider } from './new_job_caps';
 import { newJobChartsProvider, topCategoriesProvider } from './new_job';
 import { modelSnapshotProvider } from './model_snapshots';
 import type { MlClient } from '../../lib/ml_client';
+import type { AlertsClient } from '../../../../alerting/server';
 
-export function jobServiceProvider(client: IScopedClusterClient, mlClient: MlClient) {
+export function jobServiceProvider(
+  client: IScopedClusterClient,
+  mlClient: MlClient,
+  alertsClient?: AlertsClient
```

Comment:
`jobServiceProvider` is also shared by our setup contract
https://github.com/elastic/kibana/blob/02c68691263615ba89280c4c371f58c986c58fee/x-pack/plugins/ml/server/shared_services/providers/job_service.ts#L32
Although this will still work currently because `alertsClient` is optional, it might be worth adding the `alertsClient` to there so consuming plugins can also get this count.
https://github.com/elastic/kibana/blob/02c68691263615ba89280c4c371f58c986c58fee/x-pack/plugins/ml/server/shared_services/shared_services.ts#L121

Can be looked at in a follow up PR.